src/View/Helper/AssetMixHelper.php

```diff
@@ -38,7 +39,9 @@ public function css(string $path, array $options = []): ?string
         $url = $this->Url->css($path, $options);
 
         // Pass proper filename with path to mix common function
-        $mixPath = (new Mix())($url);
+        $mixPath = (new Mix(
```

Comment:
Require unit tests for this

Reply:
Have removed the dependency on Request and add some unit tests